ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_switzerland_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTcxNjcyMDU2MS4xNzQ2MzA5MjE2In0sImFub25JZCI6ImUxMDYzOTZkMWQxMWM2YzgifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouY2gvaXQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvc2Fsb29uL2MtY2xhc3Mvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjAiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI1NzQ3MjcxODEyNjE0NjQ4IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x558f9a8ff75a <unknown>
#1 0x558f9a3b24b0 <unknown>
#2 0x558f9a398d86 <unknown>
#3 0x558f9a39769a <unknown>
#4 0x558f9a397dff <unknown>
#5 0x558f9a397d54 <unknown>
#6 0x558f9a3b669f <unknown>
#7 0x558f9a45b92a <unknown>
#8 0x558f9a429592 <unknown>
#9 0x558f9a44f5e0 <unknown>
#10 0x558f9a429363 <unknown>
#11 0x558f9a3f5d63 <unknown>
#12 0x558f9a3f69c1 <unknown>
#13 0x558f9a8c4a6b <unknown>
#14 0x558f9a8c8951 <unknown>
#15 0x558f9a8acb62 <unknown>
#16 0x558f9a8c94c4 <unknown>
#17 0x558f9a89113f <unknown>
#18 0x558f9a8ed6f8 <unknown>
#19 0x558f9a8ed8d6 <unknown>
#20 0x558f9a8fe5a6 <unknown>
#21 0x7fe17e89caa4 <unknown>
#22 0x7fe17e929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_switzerland_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAyMDYwIiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJpdF9DSCIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LmNoL2l0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9jLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYwIn0sImZsYWdzIjp7InBhZ2VWaWV3Ijp0cnVlfSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiIxNzE2NzIwNTYxLjE3NDYzMDkyMTYifSwiYW5vbklkIjoiZTEwNjM5NmQxZDExYzZjOCJ9LCJwZXJmb3JtYW5jZSI6e30sImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX0sImNhdGFsb2ciOnsiUHJvZHVjdCI6eyJfaWQiOiIyMDYwIiwiZGltZW5zaW9ucyI6eyJCYXVtdXN0ZXJTaG9ydCI6WyIyMDYwIl19fX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiMTk1MDI4Mzc0NzE4ODc1MzUifQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x558f9a8ff75a <unknown>
#1 0x558f9a3b24b0 <unknown>
#2 0x558f9a398d86 <unknown>
#3 0x558f9a39769a <unknown>
#4 0x558f9a397dff <unknown>
#5 0x558f9a397d54 <unknown>
#6 0x558f9a3b669f <unknown>
#7 0x558f9a45b92a <unknown>
#8 0x558f9a429592 <unknown>
#9 0x558f9a44f5e0 <unknown>
#10 0x558f9a429363 <unknown>
#11 0x558f9a3f5d63 <unknown>
#12 0x558f9a3f69c1 <unknown>
#13 0x558f9a8c4a6b <unknown>
#14 0x558f9a8c8951 <unknown>
#15 0x558f9a8acb62 <unknown>
#16 0x558f9a8c94c4 <unknown>
#17 0x558f9a89113f <unknown>
#18 0x558f9a8ed6f8 <unknown>
#19 0x558f9a8ed8d6 <unknown>
#20 0x558f9a8fe5a6 <unknown>
#21 0x7fe17e89caa4 <unknown>
#22 0x7fe17e929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_switzerland_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjE3MTY3MjA1NjEuMTc0NjMwOTIxNiJ9LCJhbm9uSWQiOiJlMTA2Mzk2ZDFkMTFjNmM4In0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LmNoL2l0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9jLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYwIiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiNjE0NTg5MzYzMTUwNzQzMyIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x558f9a8ff75a <unknown>
#1 0x558f9a3b24b0 <unknown>
#2 0x558f9a398d86 <unknown>
#3 0x558f9a39769a <unknown>
#4 0x558f9a397dff <unknown>
#5 0x558f9a397d54 <unknown>
#6 0x558f9a3b669f <unknown>
#7 0x558f9a45b92a <unknown>
#8 0x558f9a429592 <unknown>
#9 0x558f9a44f5e0 <unknown>
#10 0x558f9a429363 <unknown>
#11 0x558f9a3f5d63 <unknown>
#12 0x558f9a3f69c1 <unknown>
#13 0x558f9a8c4a6b <unknown>
#14 0x558f9a8c8951 <unknown>
#15 0x558f9a8acb62 <unknown>
#16 0x558f9a8c94c4 <unknown>
#17 0x558f9a89113f <unknown>
#18 0x558f9a8ed6f8 <unknown>
#19 0x558f9a8ed8d6 <unknown>
#20 0x558f9a8fe5a6 <unknown>
#21 0x7fe17e89caa4 <unknown>
#22 0x7fe17e929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_switzerland_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFBpUGFnZSI6eyJ2ZWhpY2xlUEkiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LmNoL2l0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9jLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ2ZWhpY2xlQ29kZSI6IncyMDYiLCJ2ZWhpY2xlQk0iOiIyMDYwIiwidmVoaWNsZVBJSGlzdG9yeSI6IjIwNjAiLCJ0aW1lc3RhbXAiOjE3NDYzMDkyMjE3MDMsImV4cGlyYXRpb24iOjEyMH19fSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiIxNzE2NzIwNTYxLjE3NDYzMDkyMTYifSwiYW5vbklkIjoiZTEwNjM5NmQxZDExYzZjOCJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5jaC9pdC9wYXNzZW5nZXJjYXJzL21vZGVscy9zYWxvb24vYy1jbGFzcy9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MCIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sIl90b29sc0V2ZW50TGlua0lkIjoiNjcxNTU1MjUzMjU4MzU5NiIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x558f9a8ff75a <unknown>
#1 0x558f9a3b24b0 <unknown>
#2 0x558f9a398d86 <unknown>
#3 0x558f9a39769a <unknown>
#4 0x558f9a397dff <unknown>
#5 0x558f9a397d54 <unknown>
#6 0x558f9a3b669f <unknown>
#7 0x558f9a45b92a <unknown>
#8 0x558f9a429592 <unknown>
#9 0x558f9a44f5e0 <unknown>
#10 0x558f9a429363 <unknown>
#11 0x558f9a3f5d63 <unknown>
#12 0x558f9a3f69c1 <unknown>
#13 0x558f9a8c4a6b <unknown>
#14 0x558f9a8c8951 <unknown>
#15 0x558f9a8acb62 <unknown>
#16 0x558f9a8c94c4 <unknown>
#17 0x558f9a89113f <unknown>
#18 0x558f9a8ed6f8 <unknown>
#19 0x558f9a8ed8d6 <unknown>
#20 0x558f9a8fe5a6 <unknown>
#21 0x7fe17e89caa4 <unknown>
#22 0x7fe17e929c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Allure-Testing/QA-Allure-Testing/Tests/BFV1_attempt_2.png'